board/views.py

```diff
@@ -1,25 +1,25 @@
 import datetime
+import gamestore.settings as settings
 from django.shortcuts import render
 from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
-from gamestore.settings import BEARER_TOKEN, API_CLIENT_ID, API_SECRET_KEY
-from .apiwrappers import TwitterWrapper, IgdbWrapper
+from .api import igdbapi, twitterapi
 
 
-twitter_wrapper = TwitterWrapper(BEARER_TOKEN)
-igdb_wrapper = IgdbWrapper(API_CLIENT_ID, API_SECRET_KEY)
+twitter_wrapper = twitterapi.TwitterWrapper(settings.API_TWITTER_TOKEN)
+igdb_wrapper = igdbapi.IgdbWrapper(settings.API_IGDB_CLIENT_ID, settings.API_IGDB_TOKEN)
 
 
 class Game:
```

Comment:
You can move this class to a separate module since it is not a view, I think a proper module would be `board/logic/game.py`.